wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -69,6 +69,23 @@ Evaluators who show proficiency in the advisory tasks outlined above, to the poi
 
 Before adding a new advisor, the addition is sanity checked with the rest of the NAT in case of objections. If no issues arise, the new advisor will then be subject to activity expectations depending on their specific situation. If the contribution of an advisor is considered lacklustre or unnecessary at some later point, again as judged by the designated member, then they will be moved back to being an evaluator.
 
+## NAT Leadership
+
+As of March 2023, the NAT has decided to re-adopt the concept of [leadership](/wiki/Modding/QAT_Leaders), which was previously used in the [QAT](/wiki/Modding/Quality_Assurance_Team). NAT leaders have the combined responsibility of both evaluators and advisors so they can watch over and stay involved with every aspect of the NAT.
+
+The current NAT leaders are ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) and ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099).
+
+### Responsibilities
+
+NAT leaders aim to cover the following tasks:
+
+- **Activity management:** Constantly monitoring the activity of other NAT members and handling cases of low activity, either through regular checkups or through exit interviews, in case the activity issues of a certain member are problematic.
```

Comment:
```suggestion
- **Activity management:** Constantly monitoring the activity of other NAT members and handling cases of low activity, either through regular checkups or through exit interviews, in case case of activity issues regarding certain members.
```